3001-4000/LIT3956Homily.xml

```diff
@@ -6,7 +6,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <titleStmt>
             <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘደረሰ፡ አብ፡ ቅዱስ፡ አባ፡ ሳዊሮስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ፈይም፡</title>
             <title xml:lang="gez" corresp="#t1" type="normalized">Dǝrsān za-darasa ʾab qǝddus ʾabbā Sāwiros ʾeṗṗis qoṗos za-hagara Fayyǝm</title>
-            <title xml:lang="en" corresp="#t1">Homily by Severus of Fayyum on the glorious angel</title>
+            <title xml:lang="en" corresp="#t1">Homily by Severus of Fayyum on the glorious angel Gabriel</title>
```

Comment:
take [on the glorious angel Gabriel] in brackets (as not present in the provided Ethiopic), or t2, as not a direct translation? or is an approximation OK?

Reply:
yes, t2 is probably better!